build/tmp_dir.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
```

Comment:
what is this for?

Reply:
This behavior has changed after clarification from @justinvp, this file has been deleted and we just put our nuget files in  `sdk/dotnet/.nuget`.